### PR TITLE
Fix useragent in stackdriver access log

### PR DESCRIPTION
**Pull request by gargnupur**
Signed-off-by: gargnupur <gargnupur@google.com>

**What this PR does / why we need it**:
Fix useragent in stackdriver access log -> was using wrong CEL name..

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by kyessenov — APPROVED** [review]

**Review by bianpengyuan — APPROVED** [review]

**Comment by bianpengyuan** [comment]
Can we cherry-pick this to older releases?

**Comment by gargnupur** [comment]
> Can we cherry-pick this to older releases?

Added for 1.7 and 1.6..

File: extensions/common/context.cc [hunk 1]
```diff
@@ -414,7 +414,7 @@ void populateExtendedHTTPRequestInfo(RequestInfo* request_info) {
   populateExtendedRequestInfo(request_info);
 
   getValue({"request", "referer"}, &request_info->referer);
-  getValue({"request", "user_agent"}, &request_info->user_agent);
+  getValue({"request", "useragent"}, &request_info->user_agent);
```
Comment (douglas-reid) [inline extensions/common/context.cc:417]: @gargnupur @kyessenov is there some sort of linter we could write that would catch these issues at review time?

Comment (gargnupur) [reply extensions/common/context.cc:417]: Yeah something that will sync with source/extensions/filters/common/expr/context.h, to make sure we are using the right fields...

Comment (kyessenov) [reply extensions/common/context.cc:417]: Yeah, although here specifically it's using ABI, not CEL. ABI usage might be hard to statically verify.

**Pull request merged**